docs/packages/omc3/workflow.md

```diff
@@ -1,7 +1,141 @@
+Using the converter to make a compatible `SDDS` file then goes as:
+```bash
+python -m omc3.tbt_converter \
+    --files trackone \
+    --tbt_datatype trackone \
+    --outputdir .
+```
+
+Other actions are supported, such as adding noise to the data or averaging across particles.
+To use them, refer to the [converter's API documentation][tbt_converter].
+
+The converter will create new files with the `.sdds` suffix appended to the original filename.
+In our case, a `trackone.sdds` file will be created.
+
+## Frequency Analysis
+
+Once measurement or simulation is in the appropriate format, the first step as seen in the table above consists in a harmonic analysis of the data.
+To do so, `omc3` provides the `hole_in_one` entrypoint script, which will perform harmonic analysis of the data when provided with the `--harpy` flag.
+
+The script provides options involved in both data cleaning and parameter tweaking for the harmonic analysis, which is useful when you have relevant information about your measurements. 
+To use these, refer to the `Harpy Kwargs` in the [hole_in_one API documentation][hole_in_one_harpy].
+
+In our example we will leave these to their default values to keep the analysis simple, and that `harpy` outputs all the computed results.
+Running the frequency analysis then goes as:
+```bash
+python -m omc3.hole_in_one --harpy \
+    --files trackone.sdds \
+    --tbt_datatype lhc \
+    --turns 0 1023 \
+    --autotunes transverse \
+    --to_write lin spectra full_spectra bpm_summary \
+    --outputdir harpy_output
+```
+
+In the output directory, `harpy` will create TFS files with the results of the analysis for both BPMs good and identified bad BPMs.
+The filenames are determined by appending the appropriate suffixes to the entry files.
+
+!!! info "Bad BPMs"
+    BPMs are determined as bad or not depending on several options from the cleaning phase.
+    While these are given sensible defaults, one might need to tweak them manually depending on their measurement. 
+    Additionally, known bad BPMs can be given with the `--bad_bpms` flag.
+
+In the ooutput files, various properties are given in column form for each observation point.
+Running `ls harpy_output/` yields the following result:
+```bash
+trackone.sdds.ampsx		    trackone.sdds.freqsx
+trackone.sdds.ampsy		    trackone.sdds.freqsy
+trackone.sdds.bad_bpms_x	trackone.sdds.linx
+trackone.sdds.bad_bpms_y	trackone.sdds.liny
+```
+
+??? question "Column Nomenclature"
+    TODO: add a reference with the meaning of each column.
+
+## Creating a Model
+
+In order to perform the optics analysis, one needs a model of the given machine to compare to.
+For this, `omc3` provides the `model_creator` entrypoint script, which allows you to run a model simulation of the desired machine and output the needed files.
+
+??? example "Supported Machines"
+    The supported machines for model creation are `lhc`, `ps`, `esrf`, `psbooster`, `skekb`, `JPARC`, `petra` and `iota`.
```

Comment:
i'm pretty sure, `esrf`, `skekb`, `petra`, and `iota` don't have model creators ..
all of them have accelerator classes, but not necessarily every accel class has a model creator linked to it

Reply:
Resolved with f65d02c